hack/lint-test-cleanup-label.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright the KubeVirt Authors.
+#
+#
+
+EXCLUDE_PATTERN="./decorators/decorators.go|\
+./tests_suite_test.go|\
+./tests/virtctl|\
+./tests/network"
+
+if grep -rl 'OncePerOrderedCleanup' tests --include=*.go |
```

Comment:
Forgot to add `./` before the tests dir to ensure the output returns `./` before the paths as explained in above comment.

```suggestion
if grep -rl 'OncePerOrderedCleanup' ./tests --include=*.go |
```

Other than that, looks awesome to me. Thanks a lot @nirdothan!
@dhiller or @brianmcarey, do you mind also taking a look?

Reply:
Interesting.. It shows different outputs for me.
Thanks anyways!